Compile errors of user of @aws-crypto/material-management with TypeScript 3.7.2

With @aws-crypto/client-node 1.0.1, TypeScript 3.7.2, on NodeJS 12.13.0 I'm getting this error when compiling our own sources:

~~~~
> tsc && tsc -p tsconfig.es6.json

node_modules/@aws-crypto/material-management-node/build/main/node_cryptographic_materials_manager.d.ts:3:21 - error TS2440: Import declaration conflicts with local declaration of 'NodeEncryptionMaterial'.

3 NodeAlgorithmSuite, NodeEncryptionMaterial, NodeDecryptionMaterial, KeyringNode, GetEncryptionMaterials, GetDecryptMaterials } from '@aws-crypto/material-management';
                      ~~~~~~~~~~~~~~~~~~~~~~

node_modules/@aws-crypto/material-management-node/build/main/node_cryptographic_materials_manager.d.ts:3:45 - error TS2440: Import declaration conflicts with local declaration of 'NodeDecryptionMaterial'.

3 NodeAlgorithmSuite, NodeEncryptionMaterial, NodeDecryptionMaterial, KeyringNode, GetEncryptionMaterials, GetDecryptMaterials } from '@aws-crypto/material-management';
                                              ~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors.
~~~~

`tsconfig.json`:
```json
{
	"extends": "./tsconfig.base.json",
	"compilerOptions": {
		"outDir": "dist"
	}
}
```

`tsconfig.es6.json`:
```json
{
	"extends": "./tsconfig.base.json",
	"compilerOptions": {
		"module": "es2015",
		"outDir": "dist.es6"
	}
}
```

`tsconfig.base.json`:
```json
{
	"extends": "tsconfig-collaborne",
	"compilerOptions": {
		"declaration": true
	},
	"include": [
		"src"
	]
}
```

(https://github.com/Collaborne/tsconfig-collaborne/blob/master/tsconfig.json has "tsconfig-collaborne")

--- a/modules/material-management-browser/src/browser_cryptographic_materials_manager.ts
+++ b/modules/material-management-browser/src/browser_cryptographic_materials_manager.ts
@@ -16,7 +16,6 @@
 import {
   WebCryptoMaterialsManager, EncryptionRequest, // eslint-disable-line no-unused-vars
   DecryptionRequest, EncryptionContext, // eslint-disable-line no-unused-vars
-  EncryptionMaterial, DecryptionMaterial, // eslint-disable-line no-unused-vars
   WebCryptoAlgorithmSuite, WebCryptoEncryptionMaterial,
   WebCryptoDecryptionMaterial, SignatureKey, needs, readOnlyProperty,
   VerificationKey, AlgorithmSuiteIdentifier, immutableBaseClass,
@@ -29,8 +28,6 @@ import { fromBase64, toBase64 } from '@aws-sdk/util-base64-browser'
 
 export type WebCryptoEncryptionRequest = EncryptionRequest<WebCryptoAlgorithmSuite>
 export type WebCryptoDecryptionRequest = DecryptionRequest<WebCryptoAlgorithmSuite>
-export type WebCryptoEncryptionMaterial = EncryptionMaterial<WebCryptoAlgorithmSuite>
-export type WebCryptoDecryptionMaterial = DecryptionMaterial<WebCryptoAlgorithmSuite>
 export type WebCryptoGetEncryptionMaterials = GetEncryptionMaterials<WebCryptoAlgorithmSuite>
 export type WebCryptoGetDecryptMaterials = GetDecryptMaterials<WebCryptoAlgorithmSuite>
 

--- a/modules/material-management-node/src/node_cryptographic_materials_manager.ts
+++ b/modules/material-management-node/src/node_cryptographic_materials_manager.ts
@@ -15,7 +15,6 @@
 
 import {
   NodeMaterialsManager, EncryptionRequest, DecryptionRequest, EncryptionContext, // eslint-disable-line no-unused-vars
-  EncryptionMaterial, DecryptionMaterial, // eslint-disable-line no-unused-vars
   NodeAlgorithmSuite, NodeEncryptionMaterial, NodeDecryptionMaterial, SignatureKey,
   needs, VerificationKey, AlgorithmSuiteIdentifier,
   immutableClass, readOnlyProperty, KeyringNode,
@@ -28,8 +27,6 @@ import { createECDH } from 'crypto'
 
 export type NodeEncryptionRequest = EncryptionRequest<NodeAlgorithmSuite>
 export type NodeDecryptionRequest = DecryptionRequest<NodeAlgorithmSuite>
-export type NodeEncryptionMaterial = EncryptionMaterial<NodeAlgorithmSuite>
-export type NodeDecryptionMaterial = DecryptionMaterial<NodeAlgorithmSuite>
 export type NodeGetEncryptionMaterials = GetEncryptionMaterials<NodeAlgorithmSuite>
 export type NodeGetDecryptMaterials = GetDecryptMaterials<NodeAlgorithmSuite>
 

